client/client_pool/include/client/client_pool/client_pool_config.hpp

```diff
@@ -64,6 +64,7 @@ typedef struct ConcordClientPoolConfig {
   std::string signing_key_path = "resources/signing_keys";
   std::uint32_t external_requests_queue_size = 0;
   std::uint32_t trace_sampling_rate = 0;
+  bool enable_traces_in_concord_only_env = false;
```

Comment:
Can you explain why we need this flag and why isn't it used in this change?

Reply:
It will be used in concord code for concord components only test deployments

Reply:
Removed this changes after discussion with robem.